composer.json

```diff
@@ -48,10 +48,9 @@
   "scripts": {
     "post-install-cmd": [
       "yii\\composer\\Installer::postInstall",
-      "@php tests/_phpstorm.php"
-    ],
-    "post-update-cmd": [
-      "@php tests/_phpstorm.php"
+      "@php tests/_phpstorm.php",
+      "cp -n .env-example .env",
+      "php yii generate/key"
```

Comment:
this should be moved in post-create-project, otherwise it will regen the app key each time you run composer install